main.js

```diff
@@ -7,112 +8,127 @@ const attack = function attack() {
 };
 
 const player1 = {
-	title: "player1",
-	name: "scorpion",
-	hp: 65,
-	img: "./assets/scorpion.png",
-	weapon: ["axe"],
-	attack,
+	playerID: 1,
+	title: 'player1',
+	name: 'scorpion',
+	hp: 100,
+	img: './assets/scorpion.gif',
+	weapon: ['axe'],
+	changeHP: changeHP,
+	elHP: elHP,
+	renderHP: renderHP,
+	attack: attack,
 };
 
 const player2 = {
-	title: "player2",
-	name: "subzero",
-	hp: 90,
-	img: "./assets/subzero.png",
-	weapon: ["stick"],
-	attack,
+	playerID: 2,
+	title: 'player2',
+	name: 'kitana',
+	hp: 100,
+	img: './assets/kitana.gif',
+	weapon: ['stick'],
+	changeHP: changeHP,
+	elHP: elHP,
+	renderHP: renderHP,
```

Comment:
ты можешь писать просто
```suggestion
	changeHP,
	elHP,
	renderHP,
```